### PR TITLE
[CLOUD-2706] - update to use new logging-ext module from jboss-eap-modules

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-2706
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by luck3y** [comment]
EAP 7.1 logging change.

**Comment by luck3y** [comment]
@bstansberry also ping for review :P

File: cp-overrides.yaml [hunk 1]
```diff
@@ -5,7 +5,12 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: sprint-18
+                  ref: sprint-22
+          - name: jboss-eap-modules
+            git:
+                  url: https://github.com/jboss-container-images/jboss-eap-modules
+                  ref: sprint-22
```
Comment (bstansberry) [inline cp-overrides.yaml]: I don't see any sprint-22 tag or branch in this repo.

Comment (luck3y) [reply cp-overrides.yaml]: Yep, I need to make those after the sprint builds.

I thought these would be ok in -dev until the next time we release, meh, now I'm thinking about it, we're probably just better off waiting for the actual release to update these in the -cp and -rel branches.

Comment (luck3y) [reply cp-overrides.yaml]: What I mean is, we should add as part of the release merge, to update the -cp and -rel overrides vs trying to track them with sprint builds.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**